packages/@coorpacademy-components/src/organism/review-congrats/index.native.tsx

```diff
@@ -201,7 +205,11 @@ const ReviewCongrats = (props: ReviewCongratsProps) => {
         ) : null}
         <Animated.View style={[fadeInStars.animatedStyle, translateStars.animatedStyle]}>
           <CardCongrats
-            animationUri={isRankShown ? cardCongratsStar.animationLottie.animationSrc : null}
+            animationUri={
+              !cardCongratsRank || isRankShown
```

Comment:
should we put that in a variable ?
to make it more explicit ?

Reply:
why not 👍